suite-common/thp/src/thpReducer.ts

```diff
@@ -0,0 +1,109 @@
+import { AnyAction } from '@reduxjs/toolkit';
+
+import { createReducerWithExtraDeps } from '@suite-common/redux-utils';
+import { ThpSuiteCredentials } from '@suite-common/suite-types';
+import { DEVICE, DeviceThpCredentialsChanged, UI } from '@trezor/connect';
+
+import { thpActions } from './thpActions';
+
+export const THP_BUTTON_REQUESTS_NAMES = [
+    'thp_pairing_request',
+    'thp_connection_request',
+    'thp_autoconnect_credential_request',
+];
```

Comment:
[ad comment below](https://github.com/trezor/trezor-suite/pull/19038#discussion_r2100179214)
```suggestion
] as const;
type THPButtonRequestName = (typeof THP_BUTTON_REQUESTS_NAMES)[number];
```

Reply:
fixed